tests/test_run_manifests.py

```diff
@@ -32,12 +32,17 @@ def test_main_list(self, mock_logging, *mocks):
 
         mock_logging.info.assert_has_calls(
             [
-                call("OpenSearch 1.0.0"),
-                call("OpenSearch 1.0.1"),
-                call("OpenSearch 1.1.0"),
-                call("OpenSearch 1.1.1"),
-                call("OpenSearch 1.2.0"),
-                call("OpenSearch 2.0.0"),
+                call('OpenSearch 1.0.0'),
```

Comment:
This will break every time we add a new manifest - can we change the assertion to look for at least this list, but not break if there are additional calls?